pkg/util/downloads.go

```diff
@@ -252,23 +252,7 @@ func UnTargz(tarball, target string, onlyFiles []string) error {
 		}
 
 		path := filepath.Join(target, path.Base(header.Name))
-		info := header.FileInfo()
-		if info.IsDir() {
-			if err = os.MkdirAll(path, info.Mode()); err != nil {
-				return err
-			}
-			continue
-		}
-
-		file, err := os.OpenFile(path, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, info.Mode())
-		if err != nil {
-			return err
-		}
-		defer file.Close()
```

Comment:
this defer was not called until the function ended, so each file inside the for loop was open and not closed